modules/accountplan/model/scheme.class.php

```diff
@@ -231,7 +231,7 @@ function deleteScheme($AccountPlanSchemeID) {
 
     function findScheme($fakturabankId) {
       global $_lib;
-      $q = sprintf("SELECT * FROM fakturabankscheme WHERE FakturabankRemoteSchemeID = %d LIMIT 1",
+      $q = sprintf("SELECT * FROM fakturabankscheme WHERE FakturabankSchemeID = %d LIMIT 1",
```

Comment:
Instead of this, change saving of firmaId scheme type id to fakturabankscheme.FakturabankSchemeID